pom.xml

```diff
@@ -106,7 +106,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>git-client</artifactId>
-      <version>3.7.0</version>
+      <version>3.7.1-rc2780.420cdde29962</version>
```

Comment:
Needs to be replaced with the 3.7.1 release now that testing has confirmed it is an improvement.